### ISSUE TITLE
Frequent item sets missing

### ISSUE THREAD

**Issue by michael-rapp** [michael-rapp/Apriori#1]
Junit test code is:

```java
File inputFile = new File("c:/data1.txt");
double minSupport = 0.2;
Apriori<NamedItem> apriori = new Apriori.Builder<NamedItem>(minSupport).create();
Iterator<Transaction<NamedItem>> iterator = new DataIterator(inputFile);
Output<NamedItem> output = apriori.execute(iterator);
SortedSet<ItemSet<NamedItem>> frequentItemSets = output.getFrequentItemSets();		
System.out.println("frequentItemSets.size():" + frequentItemSets.size());		
Iterator<ItemSet<NamedItem>> iteratorItemSet = frequentItemSets.iterator();

while (iteratorItemSet.hasNext()) {
    ItemSet<NamedItem> itemSet = iteratorItemSet.next();
    System.out.println("result ............."+ itemSet.toString());
}
```
data1.txt content is:
```
# Test data for the Apriori algorithm
# One transaction per line, items are separated with whitespaces

bread   butter  sugar
coffee  milk    sugar
bread   coffee  milk    sugar
coffee  milk
```
run result is :
```
result .............[milk]
result .............[coffee, milk, sugar]
result .............[bread, coffee, milk, sugar]
```

the support of bread is 2/4=0.5;
the support of coffee is 3/4=0.75;
the support of sugar  is 3/4=0.75;
the support of milk  is 3/4=0.75;

all these are greater than minSupport = 0.2 , but the result only includes "milk"

**Comment by nirehs** [comment]
mark

**Comment by michael-rapp** [comment]
Fixed in version 1.0.1.
